app/index.php

```diff
@@ -1,3 +1,172 @@
 <?php
 
-echo phpinfo();
\ No newline at end of file
+require __DIR__ . '/vendor/autoload.php';
+
+use Dotenv\Dotenv;
+use Src\controller\CategoriesController;
+use Src\controller\NewsController;
+use Src\controller\UserController;
+use Src\router\Request;
+use Src\router\Response;
+use Src\router\Router;
+use Src\utils\JwtUtil;
+
+$dotenv = new DotEnv(__DIR__);
+$dotenv->load();
+
+// $dbConnection = (new DatabaseCon())->getConnection();
+
+/**
+ * Routes part
+ * TO-DO:
+ * - refactoring moving this logic to another file
+ */
+Router::get('/', function () {
+    echo 'Hello World';
+});
+
+Router::post('/users/login', function (Request $req, Response $res) {
+    UserController::logIn($req, $res);
+});
+
+Router::post('/users/register', function (Request $req, Response $res) {
+    UserController::registerUser($req, $res);
+});
+
+/**
+ * TO-DO
+ * -implement some sort of middleware logic
+ * -refactor, authorization logic
+ */
+
+//news routes
+Router::get('/news', function (Request $req, Response $res) {
+    $token = JwtUtil::validateToken($req->getHeaders());
```

Comment:
@jjsimonAB for future reference trying to implement a middleware would be ideal but yeah this is a test version

Reply:
Yes, actually I had is as a TO-DO since it was no that important for the test purposes.